### DISCUSSION TITLE
CSS scroll-behavior

### DISCUSSION THREAD

**Discussion by UCYT5040** [tailwindlabs/tailwindcss#5995] [Ideas]
I think adding scroll behavior as a class would be nice.

**Comment by NovusOpus** [comment]
There's already a [PR](https://github.com/tailwindlabs/tailwindcss/pull/5388), Hopefully is going to be included in V3.

Update: Actually it seems like it's already merged in tailwind, but I could not find something about it in the docs, Probably going to be in the updated  V3 Docs?, Maybe someone from **Tailwind CSS Team**  can confirm.

**Reply by adamwathan** [reply]
Yep it's in there for v3! Can see some details in the alpha release notes: https://github.com/tailwindlabs/tailwindcss/releases/tag/v3.0.0-alpha.1
